src/styles/_variables.scss

```
@@ -0,0 +1,11 @@
+$textFonts: "Roboto", sans-serif;
```

```suggestion
$text-fonts: "Roboto", sans-serif;
```
назывй перменные так

---

Done ✅